packages/web-components/src/components/cbp-structured-list/cbp-structured-list.scss

```diff
@@ -87,12 +87,16 @@ cbp-structured-list {
     font-style: italic;
   }
 
+  [slot=cbp-structured-list-footer]{
```

Comment:
you don't know what exactly will be slotted here (should be the action bar), so you shouldn't be trying to set CSS on it from here. It should be up to the slotted content/component, so just remove this.